src/main/java/com/faforever/client/config/CacheConfig.java

```diff
@@ -73,6 +76,9 @@ public CacheManager cacheManager() {
         new CaffeineCache(REPLAYS_RECENT, newBuilder().expireAfterWrite(10, MINUTES).build()),
         new CaffeineCache(MAP_GENERATOR, newBuilder().expireAfterWrite(10, MINUTES).build()),
         new CaffeineCache(LEADERBOARD, newBuilder().expireAfterWrite(5, MINUTES).build()),
+        new CaffeineCache(LEAGUE, newBuilder().expireAfterWrite(10, MINUTES).build()),
+        new CaffeineCache(LEAGUE_ENTRIES, newBuilder().expireAfterWrite(1, MINUTES).build()),
```

Comment:
Why so short for the entries?

Reply:
The idea was that league entries will change all the time as people play games. The divisions for example are way more static info. If you find 1 minute excessive, I can increase it.